utils/fixed_decimal/src/decimal.rs

```diff
@@ -912,12 +912,26 @@ pub enum RoundingMode {
     /// Return an error if the number has trailing nonzero digits that need to be rounded.
     Unnecessary,
 
+    /// Round to ceil
+    Ceil,
+
+    Floor,
```

Comment:
@sffc , Shall we add a comment for each element of the `RoundingMode` enum?

Reply:
Yes; ideally each of these also has a doc comment to illustrate the behavior.